src/main.html

```diff
@@ -37,6 +37,7 @@
     <link type="text/css" rel="stylesheet" href="./css/tabs/osd.css" media="all"/>
     <link type="text/css" rel="stylesheet" href="./css/tabs/vtx.css" media="all"/>
     <link type="text/css" rel="stylesheet" href="./css/tabs/power.css" media="all"/>
+    <link type="text/css" rel="stylesheet" href="./css/tabs/popupDialog.css" media="all"/>
```

Comment:
This would no longer be needed also?

Reply:
yrk, I spend hours to make a class for general purpose, but that was too much.
Sorry for the leftover